README

```diff
@@ -1,13 +1,71 @@
-You have successfully created an Atlassian Plugin!
+- You have successfully created an Atlassian Plugin!
 
 Here are the SDK commands you'll use immediately:
 
-* atlas-run   -- installs this plugin into the product and starts it on localhost
-* atlas-debug -- same as atlas-run, but allows a debugger to attach at port 5005
-* atlas-cli   -- after atlas-run or atlas-debug, opens a Maven command line window:
+* <b> atlas-run </b>  -- installs this plugin into the product and starts it on localhost
+* <b> atlas-debug </b> -- same as atlas-run, but allows a debugger to attach at port 5005
+* <b> atlas-cli </b>  -- after atlas-run or atlas-debug, opens a Maven command line window:
```

Comment:
Looks great to me, just a suggestion, since these are cli-commands you can do
```bash
atlas-run # installs this plugin into the product and starts it on localhost
```
This way someone coming to the repo can directly get a copy button on hover.